app/templates/components/explore/side-bar.hbs

```diff
@@ -1,8 +1,11 @@
-<div class="map item">
+<div class="map item {{if isMapVisible 'mobile hidden'}}">
   {{#g-map lat=37.7833 lng=-122.4167 zoom=2 address=location class='google-maps' as |context|}}
     {{g-map-address-marker context address=location onLocationChange=(action 'onLocationChangeHandler')}}
   {{/g-map}}
 </div>
+{{#if device.isMobile}}
+  <div class="ui bottom attached button" role="button" {{action 'toggleMap'}}> {{if isMapVisible 'Show' 'Hide'}} Map </div>
```

Comment:
@Anupam-dagar I'm getting confused here. When `isMapVisible` is true then you are showing the user `Show Map`. But it doesn't make any sense. I guess you missed this?